frontend/src/container/Tracim.jsx

```diff
@@ -147,8 +147,10 @@ export class Tracim extends React.Component {
     const { props } = this
     const bell = '🔔'
     const isMainTab = this.liveMessageManager.eventSource !== null
+    const notificationOptions = { tag: 'call' }
```

Comment:
Upon reading https://developer.mozilla.org/en-US/docs/Web/API/Notification/Notification I'm realizing we need this:

```suggestion
    const notificationOptions = { tag: 'call', renotify: true }
```